crowdin_api/api_resources/bundles/tests/test_bundles_resources.py

```diff
@@ -10,15 +10,19 @@
 
 class TestBundlesResource:
     resource_class = BundlesResource
+    PROJECT_ID = 1
```

Comment:
Let's keep the old way of passing a project ID into the methods and just add a couple of new tests with the new way of initializing project ID.